samples/bootloader/src/main.c

```diff
@@ -108,7 +47,8 @@ static void boot_from(const struct fw_firmware_info *fw_info)
 	printk("Attempting to boot from address 0x%x.\n\r",
 		fw_info->firmware_address);
 
-	if (!verify_firmware(fw_info->firmware_address)) {
+	if (!bl_validate_firmware_local(fw_info->firmware_address,
```

Comment:
what does local refer to here? That it's not used as an external abi?

Reply:
Correct. Naming suggestions welcome :)